### DISCUSSION TITLE
CT Selection Query: Suggestions for Measuring 600A Current with ATM90E36 IC (Based on Datasheet)

### DISCUSSION THREAD

**Discussion by Raza854** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#162] [General]
Hello everyone,

I hope you are all doing well. I am currently working on a project that involves measuring a maximum current of **600A** using the **ATM90E36** energy metering IC. **I have designed my circuit based on the information provided in the datasheet and have implemented it accordingly.** Now, I am seeking recommendations for selecting the right Current Transformers (CTs) to achieve accurate and safe measurements within this framework.

**Here are my specific questions:**

**CT Current Rating:** I need to measure currents as high as 600A with the ATM90E36 IC, following the circuit design outlined in the datasheet. Can anyone suggest CTs with a current rating of at least 600A or higher that are suitable for this application and compatible with the ATM90E36 within the parameters provided in the datasheet?

**Accuracy Considerations:** What are the recommended accuracy levels for CTs when measuring high currents like 600A within the context of the ATM90E36 and the circuit design from the datasheet? Are there any specific accuracy classes I should be looking for to ensure compatibility with the IC and the circuit as specified in the datasheet?

**Turns Ratio:** How do I determine the appropriate turns ratio for the CTs to match them with the ATM90E36 IC and the circuit design from the datasheet? What factors should I consider in this regard to ensure accurate measurements within the framework provided in the datasheet?

**Safety Precautions:** Are there any safety precautions or considerations I should be aware of when dealing with CTs for high-current measurements with the ATM90E36 IC and the circuit design derived from the datasheet?

I would greatly appreciate any insights, experiences, or product recommendations that forum members can provide regarding CT selection for measuring 600A currents within the context of the ATM90E36 IC and the circuit design based on the datasheet. Your expertise will be invaluable in helping me make the right choice for my project, given the specific framework outlined in the datasheet.

Thank you in advance for your assistance, and I look forward to learning from your valuable input.

Best regards,
Raza Kazmi

**Comment by PandemiK911** [comment]
Hello,
There's something pretty strange in your post : one should consider a design as a whole, including CTs and/or Rogowsky coils choice. That sounds more like "I'm *trying* to design something but I still have a lot a design questions".
The Application Note 46104A from the Microchip is a really valuable document associated with the datasheet.

Regarding CT ratio : you'll have to choose the CT and calculate the burden resistor accordingly. For example, for high current CT, there's often up to 1A secondary current. The ATM90 works in the 0-720mVAC range. From this, that's simple ohm-low calculations : Rburden = 0,720V/1A = 720mohm. Pretty low and resistor power would be 0.72x1² = 720mW. Pretty high for SMD 0603 resistor but not impossible for 1206 or bigger and really easy for through-hole resistors.

About accuracy : You'll have to add accuracy of CT, accuracy of burden and accuracy of ATM90 (which is a bit less than 1% thanks to sigma-delta) and you'll have a good idea of accuracy that could be achieved.

Safety : That's very important
### NEVER clamp a CT on a live wire without pluging the secondary output

Did I mention that you should
### NEVER clamp a CT on a live wire without pluging the secondary output

This is better explained here that I could ever do it : https://docs.openenergymonitor.org/electricity-monitoring/ct-sensors/introduction.html#safety
